src/main/resources/i18n/messages_en.properties

```diff
@@ -45,6 +45,7 @@ email.notification.group.editors=Editors
 # Notification Titles (based on originating type)
 email.notification.title.attachment=The attachment "{0}" for the lecture "{1}" in the course "{2}" was changed.
 email.notification.title.file.submission.successful=Your submission for the file upload exercise "{0}" in the course "{1}" was successful.
+email.notification.title.duplicate.test.cases="{0}" in the course "{1}" has multiple test cases with the same name! This critical issue needs to be resolved as quickly as possible. Otherwise, there will be problems with creating the student results!
```

Comment:
Did you use the same string as email.notification.title.duplicate.test.cases on purpose?

Reply:
What do you mean exactly (your question confuses me a bit to be honest (this is the name of this line)? Are you asking why the messages_en.properties and messages.properties contain the same strings? (if yes, this was done like this already so I just followed the established convention)

Reply:
Yes, that was my question